docs/modules/ROOT/pages/how-to/customize-cypher.adoc

```diff
@@ -0,0 +1,302 @@
+= Customize Cypher
+
+In some cases, you may need to write highly customized queries within the Cypher Builder. For instance if you:
+
+* Need to embed Cypher strings into existing Cypher queries
+* Use Cypher Builder within a larger Cypher query
+* Need to use features that are not supported in the current version of the Cypher Builder
+* Use custom functions or procedures
+
+
+[WARNING]
+====
+Embedding custom Cypher in a query may lead to Code Injection and other security issues.  
```

Comment:
Good point, this may need a better rephrasing

More than a feature, this is a way for advanced developers who may need to manually fine-tune the queries (effectively bypassing the Cypher Builder). This has always been in the builder, as we need it in the GraphQL library. This warning intention is to let the user know that if they use this they are on their own, as they are no longer using the safety mechanisms built into the Cypher Builder and need to make sure they are writing safe Cypher.

Note that this is effectively the same as not using the Cypher Builder, so it is not more unsafe than not using the Cypher Builder